components/ca-radio-group.js

```diff
@@ -23,33 +22,38 @@ define([], () => {
          */
         render() {
 
-            if (!this.data || this.data.length === 0) return;
+            if (!this.data) return;
 
             this.destroy();
 
             const container = document.createElement('fieldset');
```

Comment:
I'll bring this up in standup tomorrow too - I would highly recommend we don't obfuscate the code, particularly as we have template strings now, the boilerplate for creating an element and all it's possible attributes etc is over the top, and it's much more readable just building the element in place.